### DISCUSSION TITLE
Can't Login, URGENT PLEASE!

### DISCUSSION THREAD

**Discussion by desiboli** [vercel/community#4766] [Account Issue] [answered]
### Question

Hi!

I have an account on vercel with email login.
The email I used is a google mail account that is connected to a domain I have moved from AWS to Vercel.
Now when the domain has been transfered to Vercel I can't receive any mails anymore until I change the DNS settings on the Vercel page. HOOOW can I login to vercel if email link is the only option while I can't recieve any emails ???????????

Please help, I've contacted both [registration@vercel.com](mailto:registration@vercel.com) and [support@vercel.com](mailto:support@vercel.com) for days now and NO ONE is answering me!

**Comment by amyegan** [comment]
Sorry you ran into this problem. If you connected your Vercel account with GitHub, GitLab, or Bitbucket, then you may be able to use one of those options to log in instead of email.

When you set up the domain, did you use an [A record](https://vercel.com/docs/projects/domains/add-a-domain#apex-domains), a [nameserver](https://vercel.com/docs/projects/domains/add-a-domain#vercel-nameservers) change, or a complete [transfer](https://vercel.com/docs/projects/domains/transfer-your-domain#transfer-a-domain-to-vercel)? If you used a A record or nameserver change then the domain could still be managed with original registrar where you purchased it. While not the ideal longterm solution, changing things with your registrar may be a way for you to temporarily change the email-related DNS records you need.

If you let me know the case number from one or both of those emails you sent, I can check on the tickets for you 🙏

**Reply by desiboli** [reply]
@amyegan thank you for the response!
No this account is only connected with email. The email is adam@pluckmarket.io

The domain was transfered completely from AWS to Vercel.

This is my case number: 00169974.

Thank you! 🙏

**Reply by desiboli** [reply]
@amyegan any news please ? 🙏

**Reply by desiboli** [reply]
@amyegan status of my case please ? It's been a week since my first message to Vercel and I literally havn't heard anything from anyone but you, and now nothing again that have helped me log in to my account. Is this normal ?

**Reply by amyegan** [reply]
Sorry for the delay. Your case is currently open and awaiting support team availability.  I know this is a frustrating situation. The team is facing an unusually high number of cases right now, so wait times are longer than normal. 

You can find more info about response times at https://vercel.com/guides/vercel-support-queue-time

**Reply by sebastianroming** [reply, marked as answer]
@desiboli Our team reached out in the mentioned support case – let's continue conversation over there so we're more efficient in helping you gain access to your account again.
